integrated_channels/integrated_channel/models.py

```diff
@@ -518,6 +525,37 @@ def incomplete_delete_transmissions(
         )
         return ContentMetadataItemTransmission.objects.filter(in_db_but_failed_to_send_query)
 
+    def mark_transmission(self, mark_for):
```

Comment:
`mark_transmission` -> `_mark_transmission` if its an internal method, given the existence of things like `mark_for_create`